public/data/kumaran.json

```
@@ -0,0 +1,28 @@
+{
+    "name": "T.kumaran",
+    "type": "personal",
+    "bio": "Open Source Enthusiast!",
+    "avatar": "https://github.com/synackred.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/synackred",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/kumaran923",
+        "icon": "twitter"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Started Freelancing,Pentesting,Bugbounty Hunting",
```

I think it'd look better like this 😅 What do you think?
```suggestion
        "title": "Started Freelancing, Pentesting, and Bugbounty Hunting",
```